library/build.gradle

```diff
@@ -1,23 +1,13 @@
-buildscript {
```

Comment:
Moved this to `/build.gradle` as the latest version of lint assumes that the buildscript is placed on the root file. Otherwise you get a Configuration with name 'compileClasspath' not found error.